.github/scripts/ci_tests.sh

```diff
@@ -11,20 +11,27 @@ TEST_RUN_E2E_TESTS=${1:-1}
 TEST_CORES=${2:-2}
 TEST_JOB_COUNT=${3:-2}
 
-echo -e "Building tests"
-cmake tests -Bbuild
+echo -e "::group::Building unit tests"
+cmake -Dbuild_ut_tests=ON -Bbuild .
```

Comment:
>Bbuild [](http://example.com/codeflow?start=27&length=6)

lets keep it at the end .... because there are many logs emitted via e2e test and its hard to see the code coverage in middle.

Reply:
![image](https://user-images.githubusercontent.com/36710865/116158683-ed892400-a6a3-11eb-95ed-9d3cb311fff2.png)

I added the `::group::` tags in the logs which group each subsection under a collapsible menu. I `rm -rf build` the build directory between the ut and e2e since each needs its own build tree and wasn't sure if that messes with the code coverage.